pkg/api/render.go

```diff
@@ -24,13 +24,13 @@ func (hs *HTTPServer) RenderToPng(c *contextmodel.ReqContext) {
 
 	queryParams := fmt.Sprintf("?%s", c.Req.URL.RawQuery)
 
-	width, err := strconv.Atoi(queryReader.Get("width", "800"))
+	width, err := strconv.Atoi(queryReader.Get("width", hs.Cfg.RendererImageWidth))
 	if err != nil {
 		c.Handle(hs.Cfg, 400, "Render parameters error", fmt.Errorf("cannot parse width as int: %s", err))
 		return
 	}
```

Comment:
I think this should be refactored to something like:
```
	width := c.QueryInt("width")
	if width == 0 {
		width = hs.Cfg.RendererDefaultImageWidth
	}
```
Because it makes more sense to have the default values as int rather than string.